### PR TITLE
mark chapter titles in v1p1

### PR DESCRIPTION
**Pull request by vipranarayan14**
I have inserted titles for all 64 titles in the PDF.

Few points to note:

- The PDF doesn't have p.479 (between PDF p.487 and p.488). So, the OCRed text also doesn't have that page.
- The title 'Other Sutra Works on Dharma' in p.260 (PDF p.269) is not numbered. So, I have not marked that title.
- Some titles have diacritics in the PDF. But they are incorrect/missing in the OCRed text. I have retained the title as it is in the OCRed text.

### CODE REVIEW

File: kANe/v1p1.md [hunk 1]
```diff
@@ -4972,6 +5018,8 @@ History of Dharmaśāstra
 
 Rg. Mantras and on pp. 1103, 1190, 1199 provides expiation (in prose) for causing the death of a cow, for one who touches or carries to tbe cemetery or cremates a suicide, and for close contact of Cāudālas with houses, earthenots and states that for children, the ayed and women the expiation is only half of what is prescribed for grown-up inen. Apurárka quotes only one verse on Yáj. III. 264-265 as mild expiation in cer tain cases of causing the death of a cow. The Mit. on Yāj. III. 30 cites a long prose passage as regards the times when a bath with clothes on is necessary. 
 
+## 22 Jatukarnya
```
Comment (vvasuki) [inline kANe/v1p1.md:5021]: Missing diacritics ? Please check pdf.

File: kANe/v1p1.md [hunk 2]
```diff
@@ -5098,6 +5148,8 @@ The Mahābhārata ascribes the view to Devala tbat in man there are three jyoti8
 
 Aparārka and the Smộticandrikā cite verses from Devala on partition, inheritance, on woman's power over stridhana. These show that Devala, the jurist, flourished about the same time as the great jurists Bșhaspati and Kätyāyana. 
 
+## 24 Paithinasi
```
Comment (vvasuki) [inline kANe/v1p1.md:5151]: Missing diacritics ? Please check pdf.

File: kANe/v1p1.md [hunk 3]
```diff
@@ -5154,6 +5208,8 @@ earliest writers and works to mention him. In the Deccani College Collection the
 
 The work does not produce the impression of being early. It is in the nature of a summary of larger works on dharma. All quotatious in Hemūdri cited from Budha are not found in the mss. 
 
+## 26 Brhaspati
```
Comment (vvasuki) [inline kANe/v1p1.md:5211]: Missing diacritics ? Please check pdf.

File: kANe/v1p1.md [hunk 4]
```diff
@@ -5390,6 +5452,8 @@ FOUNDED
 
 1917 
 
+## 30 The Smrtis
```
Comment (vvasuki) [inline kANe/v1p1.md:5455]: Missing diacritics ? Please check pdf.

File: kANe/v1p1.md [hunk 5]
```diff
@@ -8050,6 +8120,8 @@ Umāsaṁhitā 51): Varāha
 
 is described ). (upavāşa ); Brahmavai varta ( 4th khanda, pūr- | Yugadharinas- vide also vārdha 8 and 26 ); Garuda under Kulisvarūpa. 116-137, Lirga, pūrvārdha Garuda 223 ; Linga 39 ; 83-84 ; Nirada, pirvār- Matsya 141-143, 164 ; dha 17-22, 110-124 ; Närada, pūrvārdha 41; Matsya 54-80, 94-100; Skanda VI. 272 ; Vāyu I. Padma ( bhūmi 87, brah- 32 and 58. 
 
+## 34 The Yājnavalkyasmrti
```
Comment (vvasuki) [inline kANe/v1p1.md:8123]: Missing diacritics ? Please check pdf.

File: kANe/v1p1.md [hunk 6]
```diff
@@ -4864,6 +4902,8 @@ SITO
 
 11 FOUNDED 
 
+## 18 Karva and Känva
```
[the remark this answers is not in the document]
Comment (vvasuki) [reply kANe/v1p1.md:4905]: Also do a mass replace of ä with ā in all kāṇē files

**Comment by vipranarayan14** [comment]
Okay. I will fix them.

**Comment by vipranarayan14** [comment]
shall I close this and submit another PR with the corrections?

**Comment by vvasuki** [comment]
BTW, you don't have to capitalize init letters exactly - so r̥ instead of R̥ is perfectly fine.

File: kANe/v1p1.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (vipranarayan14) [reply kANe/v1p1.md:4905]: There are á, ǎ, ä also. So, I'll do this and submit a separate PR.

**Comment by vipranarayan14** [comment]
Fixed diacritics in all the titles.

**Pull request merged**